README.md

```diff
@@ -121,6 +66,20 @@ What has not been tested:
 If you try one of these untested features and it works, be sure to let
 us know (creating a github issue is probably the best way for that).
 
+## Additional Documentation
+
+### PDF/Word Documentation
+
+The `doc` directory contains [LMiC-v2.3.pdf](doc/LMiC-v2.3.pdf), which documents the library APIs and use. It's based on the original IBM documentation, but has been adapted for this version of the library. However, as this library is used for more than Arduino, that document is supplemented by practical details in this document.
+
+### Adding Bandplans
+
+There is a general framework for adding new region support. [HOWTO-ADD-REGION.md](./HOWTO-ADD-REGION.md) has step-by-step instructions for adding a region.
+
+### Known bugs and issues
+
+See the list of bugs at [mcci-catena/arduino-lmic](https://github.com/mcci-catena/arduino-lmic/issues).
+
 ## Configuration
```

Comment:
Document the new configuration features here. (and in the word file)

Reply:
Change pushed. There's an incompatibility between VS Code 1.29 and Markdown TOC 1.5.6.